core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -262,8 +273,9 @@ namespace kagome::consensus::grandpa {
     return round_state;
   }
 
-  void GrandpaImpl::executeNextRound(RoundNumber round_number) {
-    if (current_round_->roundNumber() != round_number) {
+  void GrandpaImpl::executeNextRound(
```

Comment:
Hm, why does round number not identify a round?

Reply:
It is not enough, because met case when two round with the same number but different voter sets id were existed